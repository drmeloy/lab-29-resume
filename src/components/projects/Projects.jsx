import React from 'react';
import PropTypes from 'prop-types';

const Projects = projects => {
  const projectsDisplay = projects.map(item => (
    <li>{item}</li>
  ));

  return(
    <>
      <h3>Projects</h3>
      <ul>{projectsDisplay}</ul>
    </>
  );
};

Name.PropTypes = {
  projects: PropTypes.array.isRequired
}

export default Projects;
