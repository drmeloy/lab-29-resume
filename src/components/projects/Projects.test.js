import React from 'react';
import { shallow } from 'enzyme';
import Projects from './Projects.jsx';

describe('Projects component', () => {
  it('renders a Projects component', () => {
    const wrapper = shallow(<Projects projects={[]} />);
    expect(wrapper).toMatchSnapshot();
  })
});
