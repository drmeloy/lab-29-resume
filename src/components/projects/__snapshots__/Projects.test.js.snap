// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component renders a Projects component 1`] = `
<Fragment>
  <h3>
    Projects
  </h3>
  <ul />
</Fragment>
`;
